modules/Core/src/main/java/org/terasology/rendering/nui/layers/ingame/inventory/InventoryGrid.java

```diff
@@ -108,6 +126,12 @@ public void onDraw(Canvas canvas) {
         }
     }
 
+    /**
+     * Returns preferred content size of the InventoryGrid.
```

Comment:
For this javadoc blocks and others for methods with `@Override` the parent javadoc will be added automatically. If the class makes the method more specific to where more/different doc could be used then it may still be valid to include new docs. There is also an `inheritDocs` thing floating around that'll insert the parent javadoc in a spot in your *new* javadoc so you can essentially make your stuff append. I don't recall the details exactly, but check it out and experiment :-)